unity-renderer/Assets/Scripts/MainScripts/DCL/DataStore/DataStore.cs

```diff
@@ -122,5 +123,10 @@ public class DataStore_Taskbar
         {
             public readonly BaseVariable<bool> isExploreV2Enabled = new BaseVariable<bool>(false);
         }
+
+        public class DataStore_FeatureFlag
+        {
+            public readonly BaseVariable<FeatureFlag> featureFlags = new BaseVariable<FeatureFlag>();
```

Comment:
```suggestion
            public readonly BaseVariable<FeatureFlag> flags = new BaseVariable<FeatureFlag>();
```
Maybe we can rename this to just `flags` so the syntax ends up being:
`DataStore.i.featureFlags.flags` instead of `DataStore.i.fratureFlags.featureFlags`